scraper/scraper.js

```diff
@@ -6,11 +6,12 @@
 const { readFile, readdirSync, writeFile } = require("fs");
 const path = require("path");
 const { promisify } = require("util");
-const fixURI = require("./fix-uri");
+const { CompactPrefixTree: Trie } = require("compact-prefix-tree");
 
 const readFileAsync = promisify(readFile);
 const writeFileAsync = promisify(writeFile);
 
+const URL_LIST = path.resolve(__dirname + "/spec-urls.txt");
```

Comment:
won't this be an absolute path already?

Reply:
Nope. It won't work if run the bin from "respec.org > `npm run get-xref-data`"
> Error: ENOENT: no such file or directory, open 'C:\Users\sid\Documents\GitHub\respec.org\spec-urls.txt'

Reply:
Or do you mean the `path.resolve` part?

Reply:
yeah, sorry for not being clear.

Reply:
Won't hurt to keep `path.resolve`. Supports windows style paths